readme.md

```diff
@@ -16,7 +16,8 @@ the square brackets (for multiple-choice questions).
 Using jQuery, write code that makes an AJAX get request to "http://kittengifs.com/gifs". Output to the console the total number of results you get back. (Assume the server responds with a JSON array of objects representing gifs.)
 
 Your Answer:
-```js
+```
+N/A
 
```

Comment:
Would love to see you make an attempt at this, even you don't know the syntax, psuedo code the steps.  Would recommend you review the [lesson plan](https://github.com/ga-dc/curriculum/tree/master/07-apis-express-ajax/ajax), and then come back to this.